packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -75,6 +75,21 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
   theme?: ITheme;
 
   /**
+   * Accessibile label for the checkbox.
```

Comment:
might want to indicate that by default the text for the label that's passed in will be used to label the control, so we're not implying that you're not meeting accessibility standards by not using this prop